src/pages/ParseTransaction/ParseTransaction.tsx

```diff
@@ -89,16 +89,16 @@ export function ParseTransactionPage() {
 
   const isDeposit = data ? data.some((event) => event.name.toLowerCase().includes("deposit")) : false;
 
-  const { tokensData } = useTokensDataRequest(chainId);
-  const { marketsInfoData } = useMarketsInfoRequest(chainId);
-  const { glvData } = useGlvMarketsInfo(true, {
+  const { tokensData = {} } = useTokensDataRequest(chainId);
+  const { marketsInfoData = {} } = useMarketsInfoRequest(chainId);
```

Comment:
hm the rest codebase expect either undefined or objects with full data, i'd avoid breaking this rule cos in some component there could be a wrong check for that